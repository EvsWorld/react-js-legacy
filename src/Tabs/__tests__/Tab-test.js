import React from 'react';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';

import Tab from '../Tab';

describe('Tab', () => {
	describe('on mount'), () => {
		test('render a tab',() => {
			const wrapper = shallow( <Tab value='Hello Tab!' /> );
			expect(wrapper).toMatchSnapshot();
		});

		test('All tabs should have class 'tab' ', () => {
			const wrapper = mount(<Tab/>);
			expect(wrapper.props().className).toBe('tab');
		});

		test(`click on tab should add className = 'is-active'`, () => {
			const wrapper = mount
		});
	});
});
		// test('render a tab with is-active class', () => {
// 	const wrapper = mount(<Tab />);
//
// })



			// test('Tab this.prop.isActive becomes positive when tab clicked', () => {
			// 	const tabComponent = shallow(<Tab isActive='false' />);
			// 	expect(tabComponent.text()).toEqual('false');
			// 	tabComponent.find('j')
			// 	expect(tabComponent.tex``t()).toEqual('true');
			//
			// 		});
			//
