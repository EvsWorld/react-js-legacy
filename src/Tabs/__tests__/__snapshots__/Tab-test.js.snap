// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic rendering render a tab 1`] = `
<div
  className="tab "
>
  <a
    onClick={[Function]}
  >
    <span>
      Tab Title
    </span>
  </a>
</div>
`;

exports[`render a tab 1`] = `
<div
  className="tab "
>
  <a
    onClick={[Function]}
  >
    <span>
      Tab Title
    </span>
  </a>
</div>
`;
